book/en/src/by-example/hardware_tasks.md

```diff
@@ -1,24 +1,26 @@
 # Hardware tasks
 
-At its core RTIC is using the hardware interrupt controller ([ARM NVIC on cortex-m][NVIC])
-to perform scheduling and executing tasks, and all tasks except `#[init]` and `#[idle]`
+At its core RTIC is using a hardware interrupt controller ([ARM NVIC on cortex-m][NVIC])
+to schedule and start execution of tasks. All tasks except `pre-init`, `#[init]` and `#[idle]`
```

Comment:
Did we want some kind of link for the somewhat special `pre-init`, not RTIC specific, but cortex-m-rt `#[pre_init]`?

https://docs.rs/cortex-m-rt/latest/cortex_m_rt/attr.pre_init.html